v3/src/components/graph/adornments/count/count-adornment-registration.tsx

```diff
@@ -38,7 +38,15 @@ const Controls = () => {
     const redoRemoveKey = checkBoxType === "count" ? "DG.Redo.graph.hideCount" : "DG.Redo.graph.hidePercent"
 
     const setShowAdornment = checkBoxType === "count"
-      ? () => adornment.setShowCount(checked)
+      ? () => {
+        graphModel.applyModelChange(
+          () => {
+            adornment.setShowCount(checked)
+          }, {
+            undoStringKey: undoAddKey,
+            redoStringKey: redoAddKey
+          })
+      }
       : () => handleShowPercent(adornment, checked)
```

Comment:
I think you need to handle this case as well. Something like
```suggestion
      : () => () => {
        graphModel.applyModelChange(
          () => {
            adornment.setShowPercent(checked)
            setEnablePercentOptions(checked)
          }, {
            undoStringKey: undoAddKey,
            redoStringKey: redoAddKey
          })
      }
```

Reply:
As it turned out I had forgotten to remove the applyModelChange for the "count" option.